### PR TITLE
[Signup fields II] Enable synchronization

### PR DESCRIPTION
**Pull request by gonzaloriestra**
Related to CartoDB/product#337

Sync the new fields to/from central and add them to the exportations.

:warning: **DO NOT MERGE UNTIL https://github.com/CartoDB/cartodb/pull/14963 AND https://github.com/CartoDB/cartodb-central/pull/2506  ARE DEPLOYED**

### CODE REVIEW

File: spec/services/carto/user_metadata_export_service_spec.rb [hunk 1]
```diff
@@ -144,9 +144,12 @@ def destroy_user(user = @user)
     end
 
     it 'includes all user model attributes' do
+      expected_attrs = @user.attributes.symbolize_keys.keys - [:rate_limit_id, :company_employees, :use_case] +
```
Comment (alrocar) [inline spec/services/carto/user_metadata_export_service_spec.rb]: should we check that `company_employees` and `use_case` are actually been migrated?

Comment (gonzaloriestra) [reply spec/services/carto/user_metadata_export_service_spec.rb]: If you mean to update the old values to match the current options, there's no need. I've been doing some tests with the industry field, and everything works fine with obsolete values.

Comment (alrocar) [reply spec/services/carto/user_metadata_export_service_spec.rb]: No, I didn't mean that.

This test is removing `company_employees, :use_case` from the keys to check? Is that correct? (maybe I'm missing something)

Comment (gonzaloriestra) [reply spec/services/carto/user_metadata_export_service_spec.rb]: Ahh, sorry, you are right. I removed these fields in the previous PR for the tests to pass (as the fields weren't being synced yet). But now, we should check it. I'm changing it, thanks!

**Comment by oleurud** [comment]
In order to make the acceptance, I am going to merge this one into #14966

**Comment by gonzaloriestra** [comment]
Already merged into #14966

**Pull request closed without merging**